src/mpid/ch3/channels/nemesis/netmod/tcp/tcp_utility.c

```diff
@@ -160,7 +160,7 @@ int MPID_nem_tcp_is_sock_connected(int fd)
     int rc = FALSE;
     char buf[1];
     int buf_len = sizeof(buf) / sizeof(buf[0]), error = 0;
-    size_t ret_recv;
+    ssize_t ret_recv;
```

Comment:
How about `MPI_Aint`?

Reply:
In this case we're directly using the return type specified by the function in use (`recv` and `read`). `MPI_Aint` should work too, but since we're handling the return value inside the function I think I prefer it this way.